authors/apps/bookmarks/response_messages.py

```diff
@@ -0,0 +1,10 @@
+BOOKMARK_MSGS = {
+    'BOOKMARK_SUCCESSFUL': 'Article has been added to your bookmarks',
+    'BOOKMARKS_FOUND': 'Here are your bookmarks. ',
```

Comment:
Remove whitespaces at the end of your strings

Reply:
Whitespaces have been removed